### ISSUE TITLE
Много бих искал да разбера за един Легендарен Куест за 90 лвл.. 

### ISSUE THREAD

**Issue by Mympa** [DragonFire/panda#149]
за да онагледя поста си ще ползвам няколко screen-shoota ot Blizard i wowhead:
web линка е какво е куеста,колко части има и как се минава.. там ще видите как изглеждат легендарните гърбове за всеки клас.И наистина куест линията е Легендарна. Минал съм я с 5 героя на Близард, ще се радвам да я има и тук !! 
Легендарния куест е може би най-важния сега защото наградата е гръб с 608 Итем левел, докато на 25 хероик може да земеш макс 580 ИЛ (с двата ъпграйда за 500 валори). Жалкото е че легендарния куест иска и пач 5.4 където се срешаме с сина на deathwing в действие!!
ЩЕ СЕ РАДВАМ ХОРАТА КОИТО ОТГОВАРЯТ ЗА КУЕСТОВЕТЕ ДА СЕ ПОСТАРАЯТ ДА ГО НАПРАВЯТ. 
http://www.wowhead.com/guide=1653/mists-of-pandarias-legendary-questline
незнам доколко работят райдовете засега той трябва да е само 1: Mogushan Vaults s 6 (2 части по 3 боса), като оп 5-я босс: Елегон има шанс за най красивия маунт в Пандария:
http://www.wowhead.com/item=87777   Reins of the Astral Cloud Serpent
но пада само на нормал или хероик ( не пада на Looking For Raid)

Другото нещо,което искам а питам е дали ги има World Bosses of Pandaria: за този пач те са 2:
http://www.wowhead.com/npc=60491   Sha Of Anger 
http://www.wowhead.com/npc=62346  Galleon 
питам защото и от двата има шанс да падне маунт

Ще се радвам скоро да минете и на 5.3 пач защото легендарния куест иска да се събират едни неща от Trone of the Thunder (райд на Isle of Thunder, Final Boss Ra-Den)
На 5.3 пач се добавят още 2 world bosses които имат шанс да дропнат маунтове"
http://www.wowhead.com/npc=69099  Nalak
http://www.wowhead.com/npc=69161/oondasta 
като цяло Isle of Giants  е много приятно място защото може да събирате кокали от мобовете и като съберете 9999 получавате Раптор като награда:
http://www.wowhead.com/item=94290 

Като бонус към това, докато си биете мобовете там можете да се натъкнете на:
http://www.wowhead.com/item=94295   което след 3 дни ще излюпи маунт ( 3 броя са): за всички фракции са 
http://www.wowhead.com/item=94291  Reins of the Red Primal Raptor
http://www.wowhead.com/item=94292 Reins of the Black Primal Raptor
http://www.wowhead.com/item=94293 Reins of the Green Primal Raptor

**Comment by Enragelolz** [comment]
Другото нещо,което искам а питам е дали ги има World Bosses of Pandaria: за този пач те са 2:
http://www.wowhead.com/npc=60491 Sha Of Anger 
http://www.wowhead.com/npc=62346 Galleon 
питам защото и от двата има шанс да падне маунт

Sha of Anger - za momenta go nqma
Galleon - ima go , no kato go killnesh nqma loot

**Comment by DarkAngel39** [comment]
Spawn за Sha Of Anger. https://gist.github.com/DarkAngel39/b5bc0684558a4d5348c9 Боса има и скрипт, но не мога да го тествам както трябва сам. Този би трябвало да има луут, за разлика от Galleon, който е с PL (Personal Loot) и засега само пълни DBErrors лога, заедно с още няколко подобни.

**Comment by Mympa** [comment]
sha of anger sushto e s personal loot  i vsichki bosove sa s personal loot. Kogato za purvi put biesh Sha  
ti puska edin quest item koito kaTO GO VURNESH  ti dava botushki epic nqkakvi ( ne pomnq veche) :) inache vsichki world Bossove sa s personal loot

**Comment by DarkAngel39** [comment]
Да, но засега в базата данни, луута е направен като стартие босове. Поне за разлика от Galleon, би трябвало да пусне нещо. Иначе той трябва да е spawn на много места и да е pool-нат. Но засега няма от къде да се намерят всички координати и т.н.

**Comment by Mympa** [comment]
za koordinati az shte ti gi dam skoro..igraq na blizzard. Bossa se spownva na 5 mesta v KUn-Lao !!!
eto koordinatite na Sha of Anger: http://www.wowhead.com/npc=60491
Sha e Vinagi edin na realm...ne sedi na vsichki mesta koito sa pokazani.. nie na blikzard pravim crossrealm raidove za da kilvame bossovete za mountovete ! :) I Moje da kilvash bosovete 1 put na sedmica  s geroi !
